solutions/CF Save the Nature.mdx

```diff
@@ -0,0 +1,207 @@
+---
+id: cf-1223C
+source: CF
+title: Save the Nature
+author: Jesse Choe, Kevin Sheng
+---
+
+[Official Editorial](https://codeforces.com/blog/entry/66993)
+
+Binary search on the answer for the minimum number of tickets, which we call $t$, to sell to make the total ecological contribution to at least $k$. 
+
+To check if a certain number of tickets reaches a minimal contribution of $k$, we can observe that in order to minimize the number of tickets sold, we should maximize the total revenue from selling these tickets. If, however, it is impossible to sell at least $k$ tickets after selling all tickets, then output $-1$.
```

Comment:
"we can observe that in order to minimize the number of tickets sold" is irrelevant; bsearch handles this

"sell at least $k$ tickets after selling all tickets" has a typo.